TCAT/Controllers/HomeViewController.swift

```diff
@@ -182,7 +182,7 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         locationManager.delegate = self
-        locationManager.requestWhenInUseAuthorization()
+        locationManager.requestAlwaysAuthorization()
```

Comment:
Why do we need always? Uber made headlines for only having this option, which basically lets developers potentially track you wherever. I'm very cautious about what apps I allow to do this. We should only have it if the app has live background navigation like Google Maps. We can refresh the location when the app comes into the foreground again through an AppDel function or something similar.

Reply:
Yeah i thought i needed for updating current location but just tested it and i don't so going to change back